pkg/query-service/app/traces/v4/query_builder.go

```diff
@@ -292,12 +292,30 @@ func buildTracesQuery(start, end, step int64, mq *v3.BuilderQuery, panelType v3.
 	if mq.AggregateOperator == v3.AggregateOperatorNoOp {
 		var query string
 		if panelType == v3.PanelTypeTrace {
-			withSubQuery := fmt.Sprintf(constants.TracesExplorerViewSQLSelectWithSubQuery, constants.SIGNOZ_TRACE_DBNAME, constants.SIGNOZ_SPAN_INDEX_V3_LOCAL_TABLENAME, timeFilter)
-			afterSubQuery := tracesV3.AddLimitToQuery(constants.TracesExplorerViewSQLSelectAfterSubQuery, mq.Limit)
-			if mq.Offset != 0 {
-				afterSubQuery = tracesV3.AddOffsetToQuery(afterSubQuery, mq.Offset)
+			orderBySpanCount := false
+			// Check if orderBy contains a specific reference to span_count
+			for _, order := range mq.OrderBy {
```

Comment:
make this a constant and reject the API call if there are multiple values in the API call.

Reply:
We are already breaking from the loop on the first time we see span_count 

```
if order.ColumnName == "span_count" {
  orderBySpanCount = true
  break
 }
```

Reply:
I think, you're asking to only have 1 field in the mq.OrderBy, I can do that

Reply:
@nityanandagohain This PR is not related to TQL Operators